converters/toZigbee.js

```diff
@@ -1149,7 +1149,7 @@ const converters = {
         },
     },
     thermostat_keypad_lockout: {
-        key: ['keypad_lockout'],
+        key: ['keypad_lockout', 'keypad_lockout_2'],
```

Comment:
Why was this added?

Reply:
The device AV2010/32 uses the "Level 2 lockout" value for the KeypadLockout Attribute.
(For reference: Page 396 in the ZigBee cluster specifications.)

In the toZigbee Converter only level1 is currently implemented.

Reply:
And for why I added a new expose-entry:

keypad_lockout is implemented as a binary with the values 0 and 1.

keypad_lockout2 is newly implemented as a binary with the values 0 and 2.

Reply:
A better solution is to remove the `keypad_lockout_1/2` and change the `keypad_lockout` to an enum instead (look in `exposes.js` for examples)

Reply:
You're right. I changed keypad_lockout from binary to enum and removed keypad_lockout_2.